src/pages/public_registration/PublicGeneralContent.tsx

```diff
@@ -94,63 +98,58 @@ const PublicGeneralContent = ({ registration }: PublicRegistrationContentProps)
             )}
           </Typography>
         )}
+
+        <StyledPublicDoi>
+          <PublicDoi registration={registration} />
+        </StyledPublicDoi>
```

Comment:
skulle ikke denne vært sist i `StyledGroup1`, slik at det ikke blir `1.5rem` med margin før evt npiSubject også når man ike har DOI? evt sjekke at det er en doi først, men det er vel litt mer komplisert og gjort i `PublicDoi`.
kanskje margin kan flyttes til `PublicDoi` også?

Reply:
nei, den skal være på venstresiden ifølge skissene, men NPI-fagfelt skal kanskje først ja? Det finnes ikke skisser for alle de forskjellige typene, så vet ikke helt rekkefølgen på alt...

Reply:
men ja - flytter margin til PublicDoi og setter Doi sist

Reply:
StyledGroup1 er vel venstresiden, så ja, mente egentlig at NPI kanskje skulle vært flyttet over, slik at DOI alltid er nederst.
i tillegg til at det nå alltid vil legges inn en margin selv om man ikke har DOI, slik som på bildet jeg lastet opp i andre kommentaren

Reply:
ups, mente ofc Group0, ikke Group1, så se bort fra det... 🙃